common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -672,6 +672,23 @@ private Credential getFamilyRefreshTokenForAccount(@NonNull final AccountRecord
         return result;
     }
 
+    /**
+     * Load FRTs from the cache for an account matching the homeAccountId
+     * @param homeAccountId : homeAccountId for which FRT is sought
+     * @return an FRT if available else null.
+     */
+    @Nullable
+    public RefreshTokenRecord getFamilyRefreshTokenForHomeAccountId(@NonNull final String homeAccountId) {
+        final List<AccountRecord> accountRecords = mAccountCredentialCache.getAccounts();
```

Comment:
nit: inline?

Reply:
Addressed via : https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/920/commits/b8f4d0ed5b84c37dd34def53aeada19dce3418c0